Makefile

```diff
@@ -123,7 +123,6 @@ endif
 lint:
 	golangci-lint run \
 		--verbose \
-		--build-tags "$(KUBERMATIC_EDITION)" \
```

Comment:
This would make it impossible to lint CE or EE.

Reply:
Turns out I've forgot to add ce/ee tags :D

Reply:
Fixed